### ISSUE TITLE
Question about thickness in the paper

### ISSUE THREAD

**Issue by lichen14** [SHI-Labs/SGL-Retinal-Vessel-Segmentation#3]
Thanks for sharing.
I have studied the MICCAI paper, but still have doubts about the thickness calculation introduced in section 2.3. 
It is mentioned in the article that s is calculated like this 
![image](https://user-images.githubusercontent.com/25859465/138876806-b47bc802-7658-41f0-b443-da50298b96e0.png)
. But in my cognition, Ir is unknown, because it is some wrong labeled vessel segments that need to be removed. 
So I guess, is it a mistake? Or should the Ir here be the known Ic or something else?

**Comment by lichen14** [comment]
Another confusion is that I can understand why “erasing some vessel labels in the training set will drastically degrade the system performance”, which you mentioned in Section 3.3. 
But, why "the SGL learning scheme overall improves the robustness on both datasets"? 
There seems no relative evidence.

Thanks  for your sharing again.

**Comment by yzhouas** [comment]
Thanks for your interest. 
(1) For the calculation of the thickness, thanks so much for pointing it out. I_r should be I_c. We will put a note to revise it later.
(2) Fig. 5 is the evidence of the improvement of performance after training the model using SGL. Both the DICE and AUC improve consistently given different erasing ratios. That's because the model can learn more information from the pseudo label and won't overfit the training dataset too much. 

Thanks so much again for your suggestions and I appreciate more discussions if you have other questions.
